test/valid-data/interface-extended-extra-props/main.ts

```diff
@@ -0,0 +1,7 @@
+export interface MyObject extends StringMap {
+    param: string
+}
+
+export interface StringMap {
```

Comment:
What’s this Map for? I don’t see it being used.

Reply:
Sorry, a little bit of context. My team and I have been maintaining our own fork of this project, but we don't have the resources to continue adding support for new Typescript features.

Yours had far surpassed ours in that regard and so I would like to switch over to using the master fork, but there were a couple test cases of ours that weren't passing. The one in this PR and the one in my other PR.

This interface is a basic version of something we're using in our projects. The main thing I'd ask is, do you agree with the schema that is being output in my test case?

Reply:
Thanks for helping get this repo up to your fork. I'm happy to help with the integration.

I just noticed that the MyObject type extends this. Sorry for not noticing this earlier.